docs/src/main/asciidoc/transaction.adoc

```diff
@@ -300,6 +300,63 @@ And the node identifier needs to be stable over the transaction manager restarts
 
 The node name identifier may be configured via the property `quarkus.transaction-manager.node-name`.
 
+[[transaction-scope]]
+== Using `@TransactionScoped` to bind CDI beans to the transaction lifecycle
+
+You can define beans that live for as long as a transaction,
+and through CDI lifecycle events perform actions when a transaction starts and ends.
+
+Just assign the transaction xref:cdi.adoc#bean-scope[scope] to such beans using the `@TransactionScoped` annotation:
+
+[source,java]
+----
+@ApplicationScoped
```

Comment:
Shouldn't this be `@TransactionScoped` instead?

Reply:
:facepalm: Yes it should. Fixed, thanks!